lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -518,14 +518,13 @@ spec = do
             ]
 
         ["network", "parameters", "--help"] `shouldShowUsage`
-            [ "Usage:  network parameters [--port INT] EPOCH_NUMBER"
+            [ "Usage:  network parameters [--port INT]"
```

Comment:
+ "View network parameters for the current epoch"

Reply:
Good idea. I've added this to the list of things to add for the next PR: https://github.com/input-output-hk/cardano-wallet/issues/1690